### PR TITLE
feat(compute): add compute consistency group create/delete samples

### PR DESCRIPTION
**Pull request by TetyanaYahodska**
## Description
Documentation - https://cloud.google.com/compute/docs/disks/async-pd/manage-consistency-groups?authuser=0#view_disks_in_a_consistency_group
Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/CreateDiskConsistencyGroup.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                  ResourcePolicy.newBuilder().getDiskConsistencyGroupPolicy())
+              .build();
+
+      Operation response =
+          regionResourcePoliciesClient.insertAsync(project, region, resourcePolicy).get();
+
+      if (response.hasError()) {
+        System.out.printf("Error creating consistency group: %s%n", response.getError());
+        return null;
+      }
```
Comment (minherz) [inline compute/cloud-client/src/main/java/compute/disks/consistencygroup/CreateDiskConsistencyGroup.java:63]: fix anti-pattern of "eating" exceptions. throw informative exception instead. no need for print to `stdout`

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/DeleteDiskConsistencyGroup.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (ResourcePoliciesClient regionResourcePoliciesClient = ResourcePoliciesClient.create()) {
+      Operation response = regionResourcePoliciesClient
+          .deleteAsync(project, region, consistencyGroupName).get();
+
+      if (response.hasError()) {
+        System.out.printf("Error deleting consistency group: %s%n", response.getError());
+        return null;
+      }
```
Comment (minherz) [inline compute/cloud-client/src/main/java/compute/disks/consistencygroup/DeleteDiskConsistencyGroup.java:53]: fix anti-pattern of "eating" exceptions. throw informative exception instead. no need for print to `stdout`

File: compute/cloud-client/src/test/java/compute/disks/ConsistencyGroupIT.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  @AfterAll
+  public static void cleanUp()
+      throws IOException, ExecutionException, InterruptedException {
+    // Delete created consistency group
+    Operation.Status status = DeleteDiskConsistencyGroup.deleteDiskConsistencyGroup(
+        PROJECT_ID, REGION, CONSISTENCY_GROUP_NAME);
+
+    assertEquals(Operation.Status.DONE, status);
```
Comment (minherz) [inline compute/cloud-client/src/test/java/compute/disks/ConsistencyGroupIT.java:63]: this is not a test. if you want to test the code sample, please create an instance of the group before the test and delete it in the test.
alternatively, mock the client library behavior.

**Review by minherz — CHANGES_REQUESTED** [review]
please review and applied changes described in the review feedback.

further, please do not submit PR for review if it fails any requested checks.

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/CreateDiskConsistencyGroup.java (hunk 1 above)
Comment (TetyanaYahodska) [reply compute/cloud-client/src/main/java/compute/disks/consistencygroup/CreateDiskConsistencyGroup.java:63]: Thank you! Fixed

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/DeleteDiskConsistencyGroup.java (hunk 2 above)
Comment (TetyanaYahodska) [reply compute/cloud-client/src/main/java/compute/disks/consistencygroup/DeleteDiskConsistencyGroup.java:53]: Thank you! Fixed

File: compute/cloud-client/src/test/java/compute/disks/ConsistencyGroupIT.java (hunk 3 above)
Comment (TetyanaYahodska) [reply compute/cloud-client/src/test/java/compute/disks/ConsistencyGroupIT.java:63]: Thank you! Created test

**Pull request closed without merging**